### ISSUE TITLE
seg fault on start up

### ISSUE THREAD

**Issue by SingingBush** [ioquake/ioq3#696]
I opened this on the Fedora bugzilla in Feb but thought to post here as nobody has responded: https://bugzilla.redhat.com/show_bug.cgi?id=2264071

essentially when starting the binary there's a seg fault:

```
$ quake3
ioq3 1.36 linux-x86_64 Jul 21 2023
Have SSE support
----- FS_Startup -----
Segmentation fault (core dumped)
```

**Comment by Calinou** [comment]
Does this occur with ioq3 compiled from source using this repository?

**Comment by SingingBush** [comment]
no, building from source and running is fine. I suspect the Fedora maintainers need to update the way the package is built. Perhaps it's building from an old commit.

**Comment by runlevel5** [comment]
@SingingBush FYI the version that Fedora shipped is so out of date and no longer work. This PR https://src.fedoraproject.org/rpms/quake3/pull-request/2 is my effort to get the Fedora package updated

**Comment by SingingBush** [comment]
Thanks

**Comment by NuclearMonster** [comment]
Looks like this issue is resolved by fedora updating their packages? I don't think we need to track it anymore here.
